manual_jb/content/analyses.md

```diff
@@ -9,32 +9,33 @@
 7. [Combined wind wave surge building loss](analyses/combined_wind_wave_surge_building_loss)
 8. [Cumulative building damage](analyses/cumulative_building_dmg)
 9. [Electric power facility damage](analyses/epf_dmg)
-10. [Electric power facility restoration](analyses/epf_restoration)
-11. [Electric power network functionality](analyses/epn_functionality)
-12. [Household-level housing sequential recovery](analyses/housing_household_recovery)
-13. [Galveston Computable General Equilibrium (CGE)](analyses/galveston_cge.md)
-14. [Housing recovery](analyses/housing_recovery)
-15. [Housing unit allocation](analyses/housingunitallocation)
-16. [Joplin Computable General Equilibrium (CGE)](analyses/joplin_cge)
-17. [Joplin empirical restoration](analyses/joplin_empirical_restoration)
-18. [Mean damage](analyses/mean_dmg)
-19. [Multi-objective retrofit optimization](analyses/multi_retrofit_optimization)
-20. [Monte Carlo failure probability](analyses/mc_failure_prob)
-21. [Network cascading interdependency functionality](analyses/nci_functionality)
-22. [Nonstructural building damage](analyses/non_structural_building_dmg)
-23. [Pipeline damage](analyses/pipeline_dmg)
-24. [Pipeline damage with repair rate](analyses/pipeline_dmg_w_repair_rate)
-25. [Pipeline functionality](analyses/pipeline_functionality)
-26. [Pipeline restoration](analyses/pipeline_restoration)
-27. [Population dislocation](analyses/populationdislocation)
-28. [Portfolio recovery](analyses/portfolio_recovery)
-29. [Residential building recovery](analyses/residential_building_recovery)
-30. [Road damage](analyses/road_dmg)
-31. [Salt Lake City Computable General Equilibrium (CGE)](analyses/slc_cge.md)
-32. [Seaside Computable General Equilibrium (CGE)](analyses/seaside_cge)
-33. [Social Vulnerability](analyses/social_vulnerability)
-34. [Tornado electric power network (EPN) damage](analyses/tornadoepn_dmg)
-35. [Transportation recovery](analyses/transportation_recovery)
-36. [Water facility damage](analyses/waterfacility_dmg)
-37. [Water network functionality](analyses/wfn_functionality)
-38. [Water facility restoration](analyses/water_facility_restoration)
+10. [Electric power facility repair_cost](analyses/epf_repair_cost)
```

Comment:
can you remove the underscore in the title to be consistent with the other analyses? "Electric power facility repair cost"